### ISSUE TITLE
Surfingkeys ZZ (Save session and quit) wipes session

### ISSUE THREAD

**Issue by Tanath** [brookhong/Surfingkeys#1239]
## Error details

SurfingKeys: 0.9.64

Browser: 83.0.4103.56 (Official Build) beta (64-bit), Linux

## Context

I had a very large session with several windows for organization, which is normal for me. I usually use Exit from the menu so my session restores. This time I tried `ZZ` and the only thing that restored was the last window that closed. Most of my tabs had not been visited in some time and I don't see any possible way I can recovery my lost tabs. This is a major issue for me.

**Comment by brookhong** [comment]
Does `ZR` help you to restore the last session?

**Comment by Tanath** [comment]
No, because opening Chrome removed the last session. I didn't know the session was lost until I opened it. And Chrome didn't think there was a problem because it didn't crash, it was just SK closing things until there was one window left before exiting properly.

**Comment by Tanath** [comment]
I just lost my session _again_ while getting out of vim and not noticing that Chrome was focused instead. **Please** make this exit properly and save all windows, not just the last focused one!
